sunpy/instr/lyra.py

```diff
@@ -378,10 +378,10 @@ def get_lytaf_events(start_time, end_time, lytaf_path=None,
 
     Parameters
     ----------
-    start_time : `datetime.datetime` or `str`
+    start_time : `astropy.time.Time` or `str`
```

Comment:
This was discussed once during a video conference. @Cadair @nabobalis Do we want all time related variables of public functions to pass through `parse_time`?

Reply:
These parts of the docstring are supposed to specific the actual type of the input, so I would rather see them stay a list of actual linked types, rather than descriptions.

Reply:
> Do we want all time related variables of public functions to pass through parse_time?

yes, in general.


So elaborating on this a little more, I would like to bump the discussion of our docstring conventions on this to an issue. Which I will open, and then resolve out the other docstring type related comments.